packages/sms/__tests__/sms.test.ts

```diff
@@ -29,5 +29,30 @@ describe("SMS", () => {
     afterEach(function () {
         client = null
     });
+
+    test("can send SMS using JSON", async () => {
+        const resp = {
+            "message-count": "1",
+            "messages": [
+                {
+                    "to": "447700900000",
+                    "message-id": "0A0000000123ABCD1",
+                    "status": "0",
+                    "remaining-balance": "3.14159265",
+                    "message-price": "0.03330000",
+                    "network": "12345",
+                    "client-ref": "my-personal-reference",
+                    "account-ref": "customer1234"
+                }
+            ]
+        };
+        nock(BASE_URL)
+            .persist()
+            .post('/sms/json', { api_key: '12345', api_secret: 'ABCDE', from: '11155512345', to: '447700900000', text: 'Sample SMS Text' })
+            .reply(200, resp)
+
+        let results = await client.send({ to: '447700900000', from: '11155512345', text: 'Sample SMS Text' });
```

Comment:
```suggestion
        const results = await client.send({ to: '447700900000', from: '11155512345', text: 'Sample SMS Text' });
```
could probably be const